cmd/main.go

```diff
@@ -209,7 +211,13 @@ func main() {
 		if err := subnet.StartSubnetController(mgr, subnetService, subnetPortService, vpcService); err != nil {
 			os.Exit(1)
 		}
-		if err := subnetset.StartSubnetSetController(mgr, subnetService, subnetPortService, vpcService); err != nil {
+		certPath := filepath.Join(os.TempDir(), "k8s-webhook-server", "serving-certs")
```

Comment:
is this file be volume mounted?

Reply:
Yes it's mounted in ncp-deployment yaml.